src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+import os
+from qa.tasks.radosgw_admin import main
+
+from requests.auth import AuthBase
+
+from pybind.mgr.dashboard.rest_client import RestClient
```

Comment:
Absolute imports won't work most of the times.
```suggestion
from ..rest_client import RestClient
```